src/libraries/System.Formats.Cbor/ref/System.Formats.Cbor.cs

```diff
@@ -1,5 +1,6 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Looks like this line got intentionally removed, you're probably just in a torn state from an update to the generator.

So it should probably be removed from your local version manually, so we don't need further cleanup sweeps like https://github.com/dotnet/runtime/pull/38953 :smile:.

Reply:
Sorry about that.  I've merged the PR and the generator should now do the right thing.

Reply:
Oh, wait, this is genapi.  That hasn't been updated.  It'll need to be.  So, yeah, just delete this line manually.